src/libaktualizr/package_manager/CMakeLists.txt

```diff
@@ -8,9 +8,11 @@ set(HEADERS packagemanagerconfig.h
             packagemanagerfake.h
             packagemanagerinterface.h)
 
-add_library(package_manager OBJECT ${SOURCES})
+add_library(package_manager OBJECT packagemanagerfactory.cc packagemanagerfake.cc packagemanagerinterface.cc)
```

Comment:
This is a bit less than ideal; should we instead remove `packagemanagerconfig.cc` from `SOURCES`?

Reply:
ok changed